python/ray/experimental/client/common.py

```diff
@@ -27,16 +27,17 @@ def __call__(self, *args, **kwargs):
                         "Use {self._name}.remote method instead")
 
     def remote(self, *args, **kwargs):
-        if self._raylet_remote_func is not None:
-            return self._raylet_remote_func.remote(*args, **kwargs)
-        return call_remote(self, *args, **kwargs)
+        return ray.call_remote(self, *args, **kwargs)
 
     def __repr__(self):
         return "ClientRemoteFunc(%s, %s)" % (self._name, self.id)
 
     def set_remote_func(self, func):
         self._raylet_remote_func = func
 
+    def run_remote_func(self, *args, **kwargs):
+        self._raylet_remote_func.remote(*args, **kwargs)
```

Comment:
Do we need set_remote_func / run_remote_func now?

Reply:
Fair cop. Might as well just use it.